TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmodes/CurrentTele.java

```diff
@@ -3,23 +3,39 @@
 import com.qualcomm.robotcore.eventloop.opmode.OpMode;
 import com.qualcomm.robotcore.eventloop.opmode.TeleOp;
 import org.firstinspires.ftc.teamcode.hardware.Robot;
+import org.firstinspires.ftc.teamcode.util.Scheduler;
 
 import java.util.Arrays;
 
 @TeleOp(name="CurrentTele")
 public class CurrentTele extends OpMode {
-    Robot robot;
+    public Robot robot;
+    public Scheduler scheduler;
 
     @Override
     public void init() {
         robot = new Robot(hardwareMap);
+        scheduler = new Scheduler();
     }
 
     @Override
     public void loop() {
         // Drivetrain (Normal Drive)
         robot.drivetrain.telemove(0.5*(gamepad1.left_stick_y), 0.5*(gamepad1.right_stick_x));
+
+        // Turret Lift
         robot.turret.liftGrab(gamepad2.left_stick_y, gamepad2.a);
         telemetry.addData("Lift Positions", Arrays.toString(robot.turret.getPotenPos()));
+
+        boolean ring_in = false;
```

Comment:
this will never be set to true, since it is a local variable. It is also only checked once. This needs to be made into an event listener.

Reply:
Oops forgot to specify this. Should've put a comment saying that it was a dummy variable

Reply:
Got it!